ScipDotnet/Program.cs

```diff
@@ -42,7 +42,11 @@ public static async Task<int> Main(string[] args)
                 "If enabled, allow public symbol definitions to be accessible from other SCIP indexes. " +
                 "If disabled, then public symbols will only be visible within the index."),
             new Option<int>("--dotnet-restore-timeout", () => DotnetRestoreTimeout,
-                @"The timeout (in ms) for the ""dotnet restore"" command")
+                @"The timeout (in ms) for the ""dotnet restore"" command"),
+            new Option<bool>("--dotnet-skip-restore", () => false,
+                @"Skip executing ""dotnet restore"" and assume it has been run externally."),
+            new Option<FileInfo?>("--dotnet-nuget-config-file", () => null,
+                @"Provide a custom path for ""dotnet restore"" to find the NuGet.config file."),

```

Comment:
```suggestion
            new Option<FileInfo?>("--nuget-config-path", () => null,
                @"Provide a custom path for ""dotnet restore"" to find the NuGet.config file."),
```

Could you also document:
- Which paths will be inspected if this option is not provided?
- Does this handle case-insensitive matching on case-sensitive file systems (e.g. if someone is developing on both Windows and Linux, it's possible that some files have odd casing because the Windows FS is case-insensitive).

Reply:
Will update but the case sensitivity will match whatever the OS filesystem is. I can call it out as case-sensitive to be safe.